Assets/HoloToolkit/Utilities/Scripts/Editor/ProjectSettingsWindow.cs

```diff
@@ -135,6 +145,73 @@ private void UpdateSettings(BuildTarget currentBuildTarget)
                 PlayerSettings.WSA.SetCapability(PlayerSettings.WSACapability.PrivateNetworkClientServer, false);
             }
 
+            var inputManagerPath = Directory.GetParent(Path.GetFullPath(Application.dataPath)).FullName + "\\ProjectSettings\\InputManager.asset";
+            bool userPermission = Values[ProjectSetting.XboxControllerSupport];
+
+            if (userPermission)
+            {
+                userPermission = EditorUtility.DisplayDialog("Attention!",
+                    "Hi there, we noticed that you've enabled the Xbox Controller support.\n\n" +
+                    "Do you give us permission to download the latest input mapping definitions from " +
+                    "the Mixed Reality Toolkit's GitHub page and replace your project's InputManager.asset?\n\n",
```

Comment:
Is there a way to append instead of replace? I'd prefer not to be destructive here.

Reply:
Not that I'm aware of. I've kept this flexible where people don't have to use the input and map their own.